examples/step-50/step-50.cc

```diff
@@ -14,912 +14,1427 @@
  * ---------------------------------------------------------------------
 
  *
- * Author: Guido Kanschat and Timo Heister
+ * Author: Thomas C. Clevenger, Clemson University
+ *         Timo Heister, Clemson University
+ *         Guido Kanschat, Heidelberg University
+ *         Martin Kronbichler, TU Munich
```

Comment:
The corporate policy people from my university do not want me to use TU Munich but spell things out
```suggestion
 *         Martin Kronbichler, Technical University of Munich
```